docs-ref-services/postgresql.md

```diff
@@ -24,7 +24,7 @@ Learn more about [Azure Database for PostgreSQL](https://docs.microsoft.com/azur
 
 ## Client Package
 
-#### Install packages with npm
+### Install the npm package
 
 Use npm to install the PostgreSQL client module.
```

Comment:
i think this is ok, its a client module which we will sweep lter. and we can iterate